skymap_scanner/server/start_scan.py

```diff
@@ -542,40 +597,71 @@ async def serve_scan_iteration(
 
     # check if anything was actually processed
     if not pixreco_ids_sent:
-        LOGGER.info("No pixels were sent for this iteration.")
-        return 0
-    LOGGER.info(f"Done serving pixel-variations to clients: {len(pixreco_ids_sent)}.")
+        LOGGER.info("No additional pixels were sent.")
+    else:
+        LOGGER.info(f"Done serving pixels to clients: {len(pixreco_ids_sent)}.")
+    return pixreco_ids_sent
 
-    #
-    # COLLECT PIXEL-RECOS
-    #
 
+async def _serve_and_collect(
+    to_clients_queue: mq.Queue,
+    from_clients_queue: mq.Queue,
+    reco_algo: str,
+    nsides_dict: NSidesDict,
+    pixeler: PixelsToReco,
+    reporter: Reporter,
+    predictive_scanning_threshold: float,
+) -> int:
+    """Run the next (or first) scan iteration (set of pixel-recos).
+
+    Return the number of pixels sent. Stop when this is 0.
```

Comment:
`Stop when this is 0.` should be `Stop when all sent pixels have been received`?